web/wp-content/themes/dctx-v2/template-parts/content-password-protected.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-qihd7m.php, but found phpcs-scan-QIhD7M.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).